src/main/java/lotto/step2/LottoAutoApplication.java

```diff
@@ -0,0 +1,25 @@
+package lotto.step2;
+
+import lotto.step2.domain.*;
+import lotto.step2.enums.MatchNumber;
+import lotto.step2.view.InputView;
+import lotto.step2.view.ResultView;
+
+import java.util.Map;
+
+public class LottoAutoApplication {
+    public static void main(String[] args) {
+        InputView inputView = new InputView();
+        ResultView resultView = new ResultView();
+
+        LottoCount lottoCount = inputView.inputInitData();
+        LottoStrategy lottoStrategy = new LottoAutoStrategy();
+        Lottos lottos = new Lottos(lottoStrategy, lottoCount);
+
+        resultView.printLottos(lottos);
```

Comment:
```suggestion
        LottoCount lottoCount = InputView.inputInitData();
        Lottos lottos = new Lottos(new LottoAutoStrategy(), lottoCount);
        ResultView.printLottos(lottos);
```
- 뷰모델은 속성을 가지고 있지 않으니 유틸성 클래스로 작성해도 되지 않을까요

Reply:
아 속성이 따로 없고 특정 기능만 제공할떄는 유틸성으로 해도 좋을것 같네요!
감사합니다!